pkg/decorators/subscription.go

```diff
@@ -10,15 +10,15 @@ import (
 
 	"github.com/asecurityteam/awsconfig-filterd/pkg/domain"
 	"github.com/asecurityteam/awsconfig-filterd/pkg/logs"
-	"github.com/asecurityteam/components"
+	httpclient "github.com/asecurityteam/component-httpclient"
 	"github.com/asecurityteam/runhttp"
 )
 
 const typeNotification = "SubscriptionConfirmation"
 
 // SubscriptionConfig contains settings for the SubscriptionComponent.
 type SubscriptionConfig struct {
-	HTTP *components.HTTPConfig
+	HTTP *httpclient.Config
```

Comment:
Do any of the environment variables in the `docker-compose.yaml` or README need to be updated?

Reply:
No, the individual components are identical to the ones that came from the combined repo. All configuration items are the same.